snuba/clickhouse/native.py

```diff
@@ -184,12 +185,13 @@ def execute(
                     )
                     result_data: Sequence[Any]
                     trace_output = ""
-                    if capture_trace:
-                        with capture_logging() as buffer:
+                    with sentry_sdk.start_span(description=query, op="db"):
```

Comment:
```suggestion
                    with sentry_sdk.start_span(description=query, op="db.clickhouse") as span:
                        span.set_data(sentry_sdk.consts.SPANDATA.DB_SYSTEM, "clickhouse")
```

This would be helpful for us on the Performance side, for filtering!